sync2jira/main.py

```diff
@@ -299,7 +262,7 @@ def initialize_recent(config):
     :return: Nothing
     """
     # Query datagrepper
-    ret = query(category=['github', 'pagure'], delta=int(600), rows_per_page=100)
+    ret = query(category=['github'], delta=int(600), rows_per_page=100)
```

Comment:
Off topic, I admit (my apologies...), but it strikes me that, given modern machines and networking, 100 rows per page seems pretty small...especially if _Sync2Jira_ is about to get a whole lot more traffic....

Reply:
I agree, but I think that change is out of scope for this PR.

Reply:
Agreed.